clients/drcachesim/tools/view.h

```diff
@@ -147,6 +153,11 @@ class view_t : public analysis_tool_t {
     int64_t filetype_record_ord_ = -1;
     bool has_modules_;
     memtrace_stream_t *serial_stream_ = nullptr;
+
+private:
+    static constexpr int kRecordColumnWidth = 12;
```

Comment:
Hmm, I do see that a bunch of other dr$sim files are using kCamelCase for constants but I think we accidentally and incorrectly let those in, being used to other styles.  The official style from https://dynamorio.org/page_code_style.html#autotoc_md122  is all caps and some dr$sim code is using that (e.g. `constexpr int V2P_INITIAL_BITS = 9;`).  I think we have to use the all caps to follow the written-down style.  I filed #5857 on cleaning up other uses.